test/routes/routes.spec.unit.ts

```diff
@@ -118,7 +118,7 @@ describe("Basic URL Tests", () => {
       .get("/strike-off-objections/check-your-answers");
 
     expect(response.status).toEqual(200);
-    expect(response.text).toMatch(/Check your answers before submitting your objection/);
+    expect(response.text).toMatch(/Check your objection application/);
```

Comment:
Wondering why this test text changed but seemingly no corresponding text change in the view files of this PR?

Reply:
The view file was changed couple of times - test updated as well.